### PR TITLE
Fix checkstyle violations in 6 modules

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
We are getting there! #2238 

- `annotation-impl`
- `common-jpa-impl`
- `event-comment`
- `fileupload`
- `message-broker-api`
- `message-broker-impl`

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/#development-process/#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/common-jpa-impl/src/test/java/org/opencastproject/serviceregistry/impl/jpa/ServiceRegistrationJpaImplTest.java [hunk 1]
```diff
@@ -125,13 +125,13 @@ public void testQueryStatistics() throws Exception {
 
   }
 
- @Test
+  @Test
  public void testToString() throws Exception {
```
Comment (lkiesow) [inline modules/common-jpa-impl/src/test/java/org/opencastproject/serviceregistry/impl/jpa/ServiceRegistrationJpaImplTest.java]:
```suggestion
  public void testToString() throws Exception {
```

File: modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/endpoint/MessageBrokerServiceEndpoint.java [hunk 2]
```diff
@@ -39,19 +39,19 @@
...
-  property = {
+    property = {
     "service.description=Message Broker REST Endpoint",
     "opencast.service.type=org.opencastproject.message.broker.endpoint",
     "opencast.service.path=/broker"
-  },
-  immediate = true,
+    },
+    immediate = true,
   // expose interface for MH REST publisher!
```
Comment (lkiesow) [inline modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/endpoint/MessageBrokerServiceEndpoint.java]:
```suggestion
```

File: modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/impl/MessageReceiverImpl.java [hunk 3]
```diff
@@ -46,12 +46,12 @@
  * A class to receive messages from a ActiveMQ Message Broker.
  */
 @Component(
-  property = {
+    property = {
     "service.description=Message Broker Receiver",
     "service.pid=org.opencastproject.message.broker.impl.MessageReceiverImpl"
```
Comment (lkiesow) [inline modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/impl/MessageReceiverImpl.java]: Shouldn't this be indented one more level?
There are a few identical cases.

**Review by lkiesow — CHANGES_REQUESTED** [review]

File: modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/impl/MessageReceiverImpl.java (hunk 3 above)
Comment (LukasKalbertodt) [reply modules/message-broker-impl/src/main/java/org/opencastproject/message/broker/impl/MessageReceiverImpl.java]: Absolutely. So I guess I really need to pay closer attention to that and not just let my script indent everything according to how checkstyle wants it.

Unfortunately, checkstyle is still buggy when it comes to indenting these annotations. And it often suggests absolutely wrong values. So these things are not caught automatically. But yeah, gonna pay more manual attention to that from now on. And the cases in this PR should be fixed.

**Comment by LukasKalbertodt** [comment]
@lkiesow Thanks for the review. I think I fixed all your remarks.

**Pull request merged**